Avatar & alert component

Rishabh-078 opened the pull request:
Avatar of different sizes and shapes along with different types of alert messages are added.

Rishabh-078 committed:
feat: avatar and alert component added

Avatar of different sizes and shapes along with different types of alert messages are added.

Rishabh-078 committed:
docs: class name changed, style: square avatar border

Ankur9669 commented on documentation.html:
```diff
@@ -43,7 +43,111 @@ <h1>Components</h1>
+                <div class="container">
+                    <div class="avatar">
+                        <img class="ava-round-img avatar-xxl" src="./assets/avatar.png" alt="Avatar" /></div>
+                    <div class="avatar">
```
The indentation could be improved

Rishabh-078 replied:
Done

Ankur9669 commented on documentation.html:
```diff
@@ -43,7 +43,111 @@ <h1>Components</h1>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28133%2C122%2C53%2C1%29&t=night-owl&wt=sharp&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522ava-round-img%2520avatar-xxl%2522%2520src% ...
+                    style="width: 590px; height: 393px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
```
The inline styling in iframe could be removed

Rishabh-078 replied:
Thanks for bringing it to my notice, I had forgotten about this.
Had made the required changes.

Ankur9669 reviewed (approved):
Great Work!!!. The UI and the code are looking awesome. Do these changes and then you can merge it

veereshbv04 commented:
In front page add cursor:pointer for buttons

Rishabh-078 committed:
style:home bage button and iframes inline styling removed, refractor: code indentation

Changes made as pointed out in PR reviews.
Cursor: Pointer added to home page buttons
Iframe: Inline styling removed
